From 1cb023ff6a9cbaa1364d083e5d8a3b06ccdf579b Mon Sep 17 00:00:00 2001
From: jbelloua <jean-marie.bellouard@intel.com>
Date: Tue, 24 Feb 2015 13:57:27 +0100
Subject: [PATCH] add INT33E1 ACPI table (BT LNP)

BT LnP solution is used on BXT & CHT_HR

Change-Id: I8f0ddb56da888f1524d86ec7ca3445c53987110a
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1093
Signed-off-by: jbelloua <jean-marie.bellouard@intel.com>
---
 drivers/acpi/acpi_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index a399a9f..a1d3d29 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -33,6 +33,7 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 	{ "BCM4752" },
 	{ "LNV4752" },
 	{ "BCM2E1A" },
+	{ "INT33E1" },
 	{ "BCM2E39" },
 	{ "BCM2E3D" },
 	{ "BCM2E3A" },
-- 
1.9.1

